dali/test/python/test_operator_color_twist.py

```diff
@@ -52,7 +52,7 @@ def test_color_twist_vs_old():
     rand_it2 = RandomDataIterator(batch_size, shape=(1024, 512, 3))
     compare_pipelines(ColorTwistPipeline(batch_size, seed, iter(rand_it1), kind="new"),
                       ColorTwistPipeline(batch_size, seed, iter(rand_it2), kind="old"),
-                      batch_size=batch_size, N_iterations=16, eps=1)
+                      batch_size=batch_size, N_iterations=3, eps=1)
```

Comment:
As above

Reply:
I believe that 16 was chosen as an arbitrary number. Testing more iterations doesn't necessarily mean that we are doing better testing. If we wanted to increase test coverage, we should probably try with different arguments, for instance.